test/components/canReceiveAndGiveItems.ts

```diff
@@ -0,0 +1,99 @@
+import { IPokemon } from "../../src/components/Battles";
+import { IItemProbabilities } from "../../src/components/mods/RandomHeldItemsMod";
+import { IInventoryListing } from "../../src/components/menus/Items";
+import { FullScreenPokemon } from "../../src/FullScreenPokemon";
+import { it } from "../main";
+import { stubBlankGame } from "../utils/fakes";
+
+it("Pokemon can be given an item and item is updated in inventory", (): void => {
+    // Arrange
+    const fsp: FullScreenPokemon = stubBlankGame();
+    const pokemonTitle: string[] = "CHARMANDER".split("");
+    let listing: IInventoryListing = {
+        amount: 10,
+        item: "Potion"
+    };
+
+    // Act
+    const pokemon: IPokemon = fsp.equations.newPokemon({
+        level: 1,
+        title: pokemonTitle
+    });
+    pokemon.item = listing.item.split("");
```

Comment:
This isn't really testing what you say you want to test. Your test is supposed to be that creating a new Pokemon with `newPokemon` can give it an item... but this manually adds the item later.